common/engine/keyboardprocessor/tests/unit/kmx/kmx.cpp

```diff
@@ -27,7 +27,7 @@ std::string utf16_to_utf8(std::u16string utf16_string); // defined in keyboard.c
 namespace
 {
 
-const std::string base = "tests/unit/kmx/";
+const std::string base = "../tests/unit/kmx/";
```

Comment:
This makes an unwarranted assumption that you have configured meson in a directory parallel to tests. It works with this reverted.

Reply:
Yes, because I was trying to avoid the whole `copy()` issue with meson. Is there a reason we can't update meson?